src/components/screens/Cadastros/EditaisContratos/ContratosRelacionados.jsx

```diff
@@ -1,119 +1,110 @@
-import React, { Component, Fragment } from "react";
+import React, { Component, Fragment as div } from "react";
 import { Field } from "redux-form";
 import {
   LabelAndInput,
   LabelAndDate
 } from "../../../Shareable/labelAndInput/labelAndInput";
-import BaseButton, { ButtonStyle } from "../../../Shareable/button";
 import StatefulMultiSelect from "@khanacademy/react-multi-select";
 
+import { renderizarLabelLote, renderizarLabelDiretoria, renderizarLabelEmpresa } from "./helper";
+
 class ContratosRelacionados extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      quantidadeForm: 0,
-      nomeDoFormAtual: "",
-      lista_lotes: null
+      lotesSelecionados: [],
+      diretoriasSelecionadas: [],
+      empresasSelecionadas: []
     };
-    
-    this.obtemNomeForm = this.obtemNomeForm.bind(this);
-    
   }
 
-  obtemNomeForm(form, quantidadeForm) {
-    let nome = `${form}${quantidadeForm + 1}`;
-    this.setState({ nomeDoFormAtual: nome})
-    this.setState({ quantidadeForm: quantidadeForm +1 })
+  lidarComDiretoriasSelecionadas(values) {
```

Comment:
lidarComDiretoriasSelecionadas(diretoriasSelecionadas) {
    this.setState({ diretoriasSelecionadas });
  }

Reply:
atualizarDiretoriasSelecionadas(diretoriasSelecionadas) {
    this.setState({ diretoriasSelecionadas });
  }